lms/djangoapps/teams/startup.py

```diff
@@ -0,0 +1,22 @@
+"""
+Initialize the mako template lookup
+"""
+from django.conf import settings
+
+from edxmako import LOOKUP
```

Comment:
It looks like we do have custom template loaders for mako files, though I'm not sure if this is actually being used.

/edx-platform/common/djangoapps/edxmako/makoloader.py

It is configured in LMS' common.py:

# List of callables that know how to import templates from various sources.
TEMPLATE_LOADERS = (
    'edxmako.makoloader.MakoFilesystemLoader',
    'edxmako.makoloader.MakoAppDirectoriesLoader',

    # 'django.template.loaders.filesystem.Loader',
    # 'django.template.loaders.app_directories.Loader',

)